addons/l10n_se/models/res_partner.py

```diff
@@ -22,7 +22,6 @@ def compute_orgnumber(self):
 
 
 def mod10r_se(number):
-    n = len(number)
-    digits = [int(d) for d in re.sub(r'\D', '', number)][-n:]
+    digits = [int(d) for d in re.sub(r'\D', '', number)][-len(number):]
```

Comment:
I think `[int(d) for d in re.sub(r'\D', '', number)][-len(number):]` is always equal to `[int(d) for d in re.sub(r'\D', '', number)]`

Reply:
yes